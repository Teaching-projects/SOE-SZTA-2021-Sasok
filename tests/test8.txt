147
15
258
11