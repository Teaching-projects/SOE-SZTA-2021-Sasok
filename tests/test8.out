Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: 
A Csata elkezdődött! Warrior kezdheti a támadást Shaman ellen.
1. körben Warrior támad --- Warrior élete: 147 || Shaman élete: 243
2. körben Shaman támad, ---Warrior élete: 136 || Shaman élete: 243
2. körben Warrior támad --- Warrior élete: 136 || Shaman élete: 228
3. körben Shaman támad, ---Warrior élete: 125 || Shaman élete: 228
3. körben Warrior támad --- Warrior élete: 125 || Shaman élete: 213
4. körben Shaman támad, ---Warrior élete: 114 || Shaman élete: 213
4. körben Warrior támad --- Warrior élete: 114 || Shaman élete: 198
5. körben Shaman támad, ---Warrior élete: 103 || Shaman élete: 198
5. körben Warrior támad --- Warrior élete: 103 || Shaman élete: 183
6. körben Shaman támad, ---Warrior élete: 92 || Shaman élete: 183
6. körben Warrior támad --- Warrior élete: 92 || Shaman élete: 168
7. körben Shaman támad, ---Warrior élete: 81 || Shaman élete: 168
7. körben Warrior támad --- Warrior élete: 81 || Shaman élete: 153
8. körben Shaman támad, ---Warrior élete: 70 || Shaman élete: 153
8. körben Warrior támad --- Warrior élete: 70 || Shaman élete: 138
9. körben Shaman támad, ---Warrior élete: 59 || Shaman élete: 138
9. körben Warrior támad --- Warrior élete: 59 || Shaman élete: 123
10. körben Shaman támad, ---Warrior élete: 48 || Shaman élete: 123
10. körben Warrior támad --- Warrior élete: 48 || Shaman élete: 108
11. körben Shaman támad, ---Warrior élete: 37 || Shaman élete: 108
11. körben Warrior támad --- Warrior élete: 37 || Shaman élete: 93
12. körben Shaman támad, ---Warrior élete: 26 || Shaman élete: 93
12. körben Warrior támad --- Warrior élete: 26 || Shaman élete: 78
13. körben Shaman támad, ---Warrior élete: 15 || Shaman élete: 78
13. körben Warrior támad --- Warrior élete: 15 || Shaman élete: 63
14. körben Shaman támad, ---Warrior élete: 4 || Shaman élete: 63
14. körben Warrior támad --- Warrior élete: 4 || Shaman élete: 48
15. körben Shaman támad, ---Warrior élete: -7 || Shaman élete: 48

SHAMAN GYOZEDELMESKEDETT!
